src/pybind/mgr/dashboard/frontend/package.json

```
@@ -26,7 +26,8 @@
     "lint:prettier": "prettier --list-different \"{src,cypress}/**/*.{ts,scss}\"",
     "lint:html": "htmllint src/app/**/*.html && html-linter --config html-linter.config.json",
     "lint:tsc": "tsc -p src/tsconfig.app.json --noEmit && tsc -p tsconfig.spec.json --noEmit && tsc -p cypress/tsconfig.json --noEmit",
-    "lint": "npm run lint:tsc && npm run lint:tslint && npm run lint:prettier && npm run lint:html",
+    "lint:scss": "sass-lint -c ./.sasslintrc 'src/**/*.scss' -v -q",
+    "lint": "npm run lint:tsc && npm run lint:tslint && npm run lint:prettier && npm run lint:html && npm run lint:scss",
```

As the number of lining checks is increasing, we could replace the sequential run with a parallel one ([npm-run-all](https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-p.md)) and use some 'magic' to run all `lint:*` scripts:
```suggestion
    "lint": "run-p --continue-on-error --aggregate-output lint:*",
```